scss/components/modal.scss

```diff
@@ -70,13 +77,20 @@ $block: #{$fd-namespace}-modal;
     height: fd-space(9);
     @include fd-focus();
   }
+
   &__body {
     max-height: calc(100vh - #{$fd-modal-header-height} - #{$fd-modal-footer-height});
     overflow-y: auto;
-    > p {
-      padding: 0;
```

Comment:
Most default paragraphs don't have default padding—this wasn't doing anything on our playground examples.